planet/subscription_request.py

```diff
@@ -212,6 +219,86 @@ def catalog_source(
     return {"type": "catalog", "parameters": parameters}
 
 
+def planetary_variable_source(
+    var_type: Literal["biomass_proxy",
```

Comment:
Just wanted to point out that internally we are already using more sources for upcoming forest products.

I'm not sure if there is a way to not duplicate these lists.

Reply:
@cpaulik I'm going off what we have at https://developers.planet.com/docs/subscriptions/pvs-subs/#planetary-variables-types-and-ids. Do you know if we publish a machine-readable version of this table for customers?

I'm using the Literal type hints here so that developers get value hints from their IDEs. The SDK does not at this time validate the values at runtime. Thus you can use any values you want, including unavailable PV types and ids. See for example https://github.com/planetlabs/planet-client-python/pull/982/files#diff-661d60b3722e519e66594c89a655b8aa191b919ac27c98ecffde58f984ab120eR355.

Reply:
You sparked my curiosity. Since our API docs spell them out I did some digging and found that https://api.planet.com/subscriptions/v1/spec includes the sources types under `schemas.PVSourceTypes`